src/backend/src/services/change-requests.services.ts

```diff
@@ -341,13 +342,22 @@ export default class ChangeRequestsService {
           projectNumber,
           workPackageNumber
         }
+      },
+      include: {
+        changeRequests: true
       }
     });
 
     if (!wbsElement) throw new NotFoundException('WBS Element', wbsPipe({ carNumber, projectNumber, workPackageNumber }));
     if (wbsElement.dateDeleted)
       throw new DeletedException('WBS Element', wbsPipe({ carNumber, projectNumber, workPackageNumber }));
 
+    const { changeRequests } = wbsElement;
+    const nonDeletedChangeRequests = changeRequests.filter((changeRequest) => !changeRequest.dateDeleted);
+    if (!allChangeRequestsReviewed(nonDeletedChangeRequests)) {
+      throw new HttpException(400, 'Please resolve all related change requests before proceeding');
```

Comment:
ok last change I promise. Can you make the error message have the wbsNum and name as well, so the error is very explicit. Would look like this:
<img width="530" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/174d9417-6605-42c2-b171-24ca38f2caa0">